content/chef_repo.md

```diff
@@ -68,7 +68,7 @@ to contain cookbook-specific copyright, email, and license data.
 The `data_bags/` directory is used to store all of the data bags that
 exist for an organization. Each sub-directory corresponds to a single
 data bag on the Chef Infra Server and contains a JSON file for each data
-bag item. If a sub-directory does not exist, then create it using SSL
+bag item. If a sub-directory does not exist, then create it using TLS/SSL
```

Comment:
@tas50 I don't understand this--is it wrong? How does one create a data bag with SSL commands?
```suggestion
bag item. If a sub-directory does not exist, then create it using SSL
```